src/lib.rs

```diff
@@ -1,7 +1,7 @@
 #![cfg_attr(docsrs, feature(doc_cfg))]
 #![expect(
     clippy::doc_markdown,
-    reason = "Android GameActivity does not need to be code-formatted."
+    reason = "Clippy lints for un-backticked identifiers within the cargo features list, which we don't want."
```

Comment:
Notably, it *is* possible to fix this lint entirely. However, when testing it out, I found the resulting documentation a little harder to read.

Still, if removing this expectation is desired, then I can include that in the PR.